diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4c50538..67f9df3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -69,7 +69,7 @@ endif()
 #-----------------------------------------------------------------------------
 set (API_VERSION_MAJOR 4)
 set (API_VERSION_MINOR 4)
-set (API_VERSION_PATCH 2)
+set (API_VERSION_PATCH 3)
 set (API_VERSION "${API_VERSION_MAJOR}.${API_VERSION_MINOR}.${API_VERSION_PATCH}")
 set (ABI_CURRENT 1)
 set (ABI_REVISION 0)
@@ -78,7 +78,7 @@ set (ABI_VERSION "${ABI_CURRENT}.${ABI_REVISION}.${ABI_AGE}")
 
 set (NEXUS_VERSION_MAJOR 4)
 set (NEXUS_VERSION_MINOR 4)
-set (NEXUS_VERSION_PATCH 2)
+set (NEXUS_VERSION_PATCH 3)
 set (NEXUS_SONAME 4)
 set (NAPI_VERSION "${NEXUS_VERSION_MAJOR}.${NEXUS_VERSION_MINOR}.${NEXUS_VERSION_PATCH}")
 set (NXLTVERSINFO "${NEXUS_VERSION_MAJOR}:${NEXUS_VERSION_MINOR}:${NEXUS_VERSION_PATCH}")
