diff --git a/applications/NXtranslate/FRM2/CMakeLists.txt b/applications/NXtranslate/FRM2/CMakeLists.txt
index 5f7c7e5..477efe2 100644
--- a/applications/NXtranslate/FRM2/CMakeLists.txt
+++ b/applications/NXtranslate/FRM2/CMakeLists.txt
@@ -30,3 +30,5 @@ include_directories("${CMAKE_CURRENT_SOURCE_DIR}")
 
 add_library (FRM2 frm2_retriever.cpp)
 
+install (TARGETS FRM2 DESTINATION ${CMAKE_INSTALL_LIBDIR}
+         COMPONENT Runtime)
