diff --git a/applications/NXtranslate/spec/CMakeLists.txt b/applications/NXtranslate/spec/CMakeLists.txt
index a7e4489..c46d343 100644
--- a/applications/NXtranslate/spec/CMakeLists.txt
+++ b/applications/NXtranslate/spec/CMakeLists.txt
@@ -30,4 +30,5 @@ include_directories("${CMAKE_CURRENT_SOURCE_DIR}")
 
 add_library (Spec SPEClib.cpp spec_retriever.cpp)
 
-
+install (TARGETS Spec DESTINATION ${CMAKE_INSTALL_LIBDIR}
+         COMPONENT Runtime)
