--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,11 +82,6 @@
 set (NEXUS_SONAME 4)
 set (NAPI_VERSION "${NEXUS_VERSION_MAJOR}.${NEXUS_VERSION_MINOR}.${NEXUS_VERSION_PATCH}")
 set (NXLTVERSINFO "${NEXUS_VERSION_MAJOR}:${NEXUS_VERSION_MINOR}:${NEXUS_VERSION_PATCH}")
-
-#-----------------------------------------------------------------------------
-# Ensure we install the license file
-#-----------------------------------------------------------------------------
-install (FILES COPYING)
 
 #-----------------------------------------------------------------------------
 # define command line options by which the user can configure the build

