diff --git a/applications/NXtranslate/loopy/CMakeLists.txt b/applications/NXtranslate/loopy/CMakeLists.txt
index cbd9dfc..a3fe2dd 100644
--- a/applications/NXtranslate/loopy/CMakeLists.txt
+++ b/applications/NXtranslate/loopy/CMakeLists.txt
@@ -30,3 +30,5 @@ include_directories("${CMAKE_CURRENT_SOURCE_DIR}")
 
 add_library (Loopy retriever.cpp)
 
+install (TARGETS Loopy DESTINATION ${CMAKE_INSTALL_LIBDIR}
+         COMPONENT Runtime)
