diff --git a/applications/NXtranslate/esrf_edf/CMakeLists.txt b/applications/NXtranslate/esrf_edf/CMakeLists.txt
index d054b7e..545c87e 100644
--- a/applications/NXtranslate/esrf_edf/CMakeLists.txt
+++ b/applications/NXtranslate/esrf_edf/CMakeLists.txt
@@ -30,3 +30,5 @@ include_directories("${CMAKE_CURRENT_SOURCE_DIR}")
 
 add_library (Edf edf_reader.cpp edf_retriever.cpp)
 
+install (TARGETS Edf DESTINATION ${CMAKE_INSTALL_LIBDIR}
+         COMPONENT Runtime)
