diff --git a/src/ifeffit_wrap.c b/src/ifeffit_wrap.c
index 119f8c35..f16fb59e 100644
--- a/src/ifeffit_wrap.c
+++ b/src/ifeffit_wrap.c
@@ -631,7 +631,7 @@ XS(_wrap_iff_exec) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -657,7 +657,7 @@ XS(_wrap_ifeffit) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -686,7 +686,7 @@ XS(_wrap_iff_get_string) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -715,7 +715,7 @@ XS(_wrap_iff_put_string) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -747,7 +747,7 @@ XS(_wrap_iff_get_scalar) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -779,7 +779,7 @@ XS(_wrap_iff_put_scalar) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -811,7 +811,7 @@ XS(_wrap_iff_get_array) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -849,7 +849,7 @@ XS(_wrap_iff_put_array) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -875,7 +875,7 @@ XS(_wrap_iff_get_echo) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -905,7 +905,7 @@ XS(_wrap_iff_strval) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -931,7 +931,7 @@ XS(_wrap_iff_scaval) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -959,7 +959,7 @@ XS(_wrap_ifeffit_) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -992,7 +992,7 @@ XS(_wrap_iffgetstr_) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1026,7 +1026,7 @@ XS(_wrap_iffgetsca_) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1060,7 +1060,7 @@ XS(_wrap_iffgetarr_) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1100,7 +1100,7 @@ XS(_wrap_iffputarr_) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1128,30 +1128,7 @@ XS(_wrap_iffgetecho_) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
-}
-
-
-XS(_wrap_new_Pint) {
-    char _swigmsg[SWIG_MAX_ERRMSG] = "";
-    const char *_swigerr = _swigmsg;
-    {
-        int *result;
-        int argvi = 0;
-        dXSARGS;
-        
-        if ((items < 0) || (items > 0)) {
-            SWIG_croak("Usage: new_Pint();");
-        }
-        result = (int *)new_Pint();
-        
-        ST(argvi) = sv_newmortal();
-        SWIG_MakePtr(ST(argvi++), (void *) result, SWIGTYPE_p_int,0);
-        XSRETURN(argvi);
-        fail:
-        (void) _swigerr;
-    }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1176,7 +1153,7 @@ XS(_wrap_copy_Pint) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1203,7 +1180,7 @@ XS(_wrap_delete_Pint) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1232,7 +1209,7 @@ XS(_wrap_Pint_assign) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1261,7 +1238,7 @@ XS(_wrap_Pint_value) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1284,7 +1261,7 @@ XS(_wrap_new_Pdbl) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1310,7 +1287,7 @@ XS(_wrap_copy_Pdbl) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1337,7 +1314,7 @@ XS(_wrap_delete_Pdbl) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1367,7 +1344,7 @@ XS(_wrap_Pdbl_assign) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1396,7 +1373,7 @@ XS(_wrap_Pdbl_value) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1421,7 +1398,7 @@ XS(_wrap_new_Parr) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1448,7 +1425,7 @@ XS(_wrap_delete_Parr) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1479,7 +1456,7 @@ XS(_wrap_Parr_getitem) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1511,7 +1488,7 @@ XS(_wrap_Parr_setitem) {
         fail:
         (void) _swigerr;
     }
-    croak(_swigerr);
+    croak("%s", _swigerr);
 }
 
 
@@ -1557,7 +1534,7 @@ static swig_command_info swig_commands[] = {
 {"Ifeffit::iffgetarr_", _wrap_iffgetarr_},
 {"Ifeffit::iffputarr_", _wrap_iffputarr_},
 {"Ifeffit::iffgetecho_", _wrap_iffgetecho_},
-{"Ifeffit::new_Pint", _wrap_new_Pint},
+//{"Ifeffit::new_Pint", _wrap_new_Pint},
 {"Ifeffit::copy_Pint", _wrap_copy_Pint},
 {"Ifeffit::delete_Pint", _wrap_delete_Pint},
 {"Ifeffit::Pint_assign", _wrap_Pint_assign},
