--- VTK/Rendering/OpenGL/vtkOpenGL.h.install    2014-12-01 14:31:47.658538756 -0500
+++ VTK/Rendering/OpenGL/vtkOpenGL.h    2014-12-01 14:32:25.215479485 -0500
@@ -19,7 +19,7 @@
 #include "vtkConfigure.h"
 
 // To prevent gl.h to include glext.h provided by the system
-#define GL_GLEXT_LEGACY
+//#define GL_GLEXT_LEGACY
 #if defined(__APPLE__) && (defined(VTK_USE_CARBON) || defined(VTK_USE_COCOA))
 # include <OpenGL/gl.h> // Include OpenGL API.
 #else
